scripts/load_csv_prereg.py

```diff
@@ -0,0 +1,100 @@
+""" Load pre-reg questions into schema from csv file
+"""
+
+import csv
+import urllib2
+import logging
+import os
+import json
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+""" Given a question number, returns list containing page id, page index, and index of question on page
```

Comment:
Please format this file according to our [style guidelines](https://cosdev.readthedocs.org/en/latest/style_guides/python.html).